uportal-war/src/main/java/org/jasig/portal/rendering/StylesheetAttributeSource.java

```diff
@@ -133,4 +150,35 @@ public IStylesheetDescriptor getStylesheetDescriptor(HttpServletRequest request)
     }
     
     public abstract PreferencesScope getStylesheetPreferencesScope(HttpServletRequest request);
+
+    protected abstract Logger getLogger();
+
+    protected boolean shouldDoSpelEvaluationForAttributeValue(final String attributeValue) {
+        return this.portalSpELService != null && attributeValue != null && attributeValue.startsWith("${") && attributeValue.endsWith("}");
```

Comment:
Might have been cool to allow an expression such as student-${person.attributeMap['college']}-apps, but  you can handle that with SpEL so this approach is sufficient and probably cleaner.